quickwit-indexing/src/source/kinesis/api.rs

```diff
@@ -30,15 +32,22 @@ use rusoto_kinesis::{
 pub(crate) async fn get_records(
     kinesis_client: &KinesisClient,
     shard_iterator: String,
+    retry_params: &RetryParams,
```

Comment:
nit: for all those Kinesis API functions, I'd move the `retry_params` parameter to the second position so that `kinesis_client` and `retry_params` always occupy the same position.